src/analyses/varEq.ml

```diff
@@ -587,6 +587,11 @@ struct
           remove ask (Cil.var v) st
         in
         List.fold_left remove_var man.local (EscapeDomain.EscapedVars.elements vars)
+    | Events.Longjmped {lval} ->
+      BatOption.map_default (fun lv ->
+          let ask = Analyses.ask_of_man man in
+          man.local|> remove_reachable ~deep:false ask [mkAddrOf lv])
```

Comment:
This is unnecessarily roundabout: taking the address of `lv` and then `remove_reachable ~deep:false` will query for the points-to-set, which will just be `lv` itself.
I think something like this should do the same thing, but I haven't (type)-checked:
```suggestion
          remove ask lv man.local
```

Reply:
I'm not sure if this is true: Consider e.g. an lv al of the shape `*x =`? Or am I missing something?